### ISSUE TITLE
New gradle versions can't build app using jail-monkey

### ISSUE THREAD

**Issue by UgurGumushan** [GantMan/jail-monkey#274]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `jail-monkey@2.8.0` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/jail-monkey/android/build.gradle b/node_modules/jail-monkey/android/build.gradle
index 158b55d..4742858 100644
--- a/node_modules/jail-monkey/android/build.gradle
+++ b/node_modules/jail-monkey/android/build.gradle
@@ -5,6 +5,7 @@ def safeExtGet(prop, fallback) {
 apply plugin: 'com.android.library'
 
 android {
+    namespace "com.gantix.JailMonkey"
     compileSdkVersion safeExtGet('compileSdkVersion', 31)
     buildToolsVersion safeExtGet('buildToolsVersion', "31.0.0")
 
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by Naandalist** [comment]
I find this npm module has updated here ➡️ https://www.npmjs.com/package/jail-monkey-new

**Comment by kmye** [comment]
have published the latest changes in 2.8.1!
